backend/orcid.py

```diff
@@ -244,37 +248,66 @@ def fetch_and_save(self, researcher, profile=None):
 
             count += 1
 
-
-    def bulk_import(self, directory, fetch_papers=True, use_doi=False):
+    def bulk_import(self, directory, fetch_papers=True, use_doi=False,
+                    start_from=None):
         """
         Bulk-imports ORCID profiles from a dmup
         (warning: this still uses our DOI cache).
         The directory should contain json versions
         of orcid profiles, as in the official ORCID
         dump.
-        """
 
+        :param directory: The directory to load the ORCID API dump from.
+        :param start_from: An optional file name to start from (of the form
+            ``<ORCID_ID>.json``, without directories).
+        """
+        nb_profiles = 0
+        start_time = time.time()
         for root, _, fnames in os.walk(directory):
             for fname in fnames:
-                #if fname == '0000-0003-1349-4524.json':
-                #    seen = True
-                #if not seen:
-                #    continue
+                fpath = path.join(root, fname)
+
+                # Eventually skip first entries
+                if fname == start_from:
+                    seen = True
+                if start_from and not seen:
+                    continue
 
-                with open(path.join(root, fname), 'r') as f:
+                # Log progression
+                nb_profiles += 1
+                if nb_profiles % 1000 == 0:
```

Comment:
Should use https://github.com/dissemin/dissemin/blob/96b83f8332fcde1b7ff89eb49b47cb336055e226/backend/utils.py#L120 instead.